components/sync/service/brave_sync_service_impl.h

```diff
@@ -73,6 +73,9 @@ class BraveSyncServiceImpl : public SyncServiceImpl {
   void StopAndClearWithShutdownReason();
   void StopAndClearWithResetLocalDataReason();
 
+  std::string GetCustomSyncServiceURL();
```

Comment:
It looks like we can move both these methods into `BraveSyncServiceImpl` class.
So we will have all the methods right in the same place  
```
  std::string GetCustomSyncServiceURL();
  bool SetCustomSyncServiceURL(std::string custom_sync_url);

  GURL GetBraveSyncServiceURL();
  GURL GetBraveDefaultSyncServiceURL();
  ```
It will be possible to add a test case for these methods.
  
It may will require make `BraveGetSyncServiceURL` the member of `SyncServiceImpl` `chromium_src/components/sync/service/sync_service_impl.cc`.